### PR TITLE
[Bug 1166042] Some text is missing in mobile site

### PR DESCRIPTION
**Pull request by orvi2014**
[Bug 1166042] Add missing image caption to AoA mobile page

### CODE REVIEW

File: kitsune/landings/templates/landings/mobile/get-involved-aoa.html [hunk 1]
```diff
@@ -27,7 +27,13 @@
 
     <div class="centered">
       <img class="smart-width" src="{{ STATIC_URL }}sumo/img/gi-visual-aoa.png" alt="">
+      <p>
+      {% trans %}
+        1 tweet can save 1 day
+      {% endtrans %}
+    </p>
     </div>
+
```
Comment (safwanrahman) [inline kitsune/landings/templates/landings/mobile/get-involved-aoa.html]: This line might be breaking the test!

Comment (orvi2014) [inline kitsune/landings/templates/landings/mobile/get-involved-aoa.html]: okay I update it and remove the line. Is everything okay now ?

**Comment by safwanrahman** [comment]
I think you should make it like this
`      <p>
        {{ _('1 tweet can save 1 day') }}
      </p>
`

File: kitsune/landings/templates/landings/mobile/get-involved-aoa.html [hunk 2]
```diff
@@ -27,8 +27,11 @@
 
     <div class="centered">
       <img class="smart-width" src="{{ STATIC_URL }}sumo/img/gi-visual-aoa.png" alt="">
+      <p>
+      {{ _('1 tweet can save 1 day') }}
+      </p>
     </div>
-
+
```
Comment (safwanrahman) [inline kitsune/landings/templates/landings/mobile/get-involved-aoa.html]: Remove the Whitespace! Its failing Travis!

**Comment by safwanrahman** [comment]
Travis Pass! :smiley: :joy: :+1: 
I have tested this in my local and it seems to be working as expected
screen shot 2015-06-22 at 23 39 58
So I think this should get r+. @mythmon thoughts about this?

File: kitsune/sumo/static/sumo/less/mobile/main.less [hunk 3]
```diff
@@ -292,6 +292,10 @@ input[type="submit"]:visited {
 
 .centered {
   text-align: center;
+  font-size: 24px;
+  text-transform: uppercase;
+  color: #2BBAE7;
+  font-family: "Komikax";
```
Comment (rehandalal) [inline kitsune/sumo/static/sumo/less/mobile/main.less]: please alphabetize these properties

**Comment by rehandalal** [comment]
This looks good. Just the little nit over alphabetizing the css properties.

Also if you could squash the commits down to one single commit and then force push, that would be great!

Thanks!

**Comment by orvi2014** [comment]
@rehandalal Is everything okay now ? can we merge this ?

File: kitsune/landings/templates/landings/mobile/get-involved-aoa.html [hunk 4]
```diff
@@ -27,8 +27,10 @@
 
     <div class="centered">
       <img class="smart-width" src="{{ STATIC_URL }}sumo/img/gi-visual-aoa.png" alt="">
+      <p>
+      {{ _('1 tweet can save 1 day') }}
```
Comment (rehandalal) [inline kitsune/landings/templates/landings/mobile/get-involved-aoa.html]: Indent one further

File: kitsune/sumo/static/sumo/less/mobile/main.less [hunk 5]
```diff
@@ -291,7 +291,11 @@ input[type="submit"]:visited {
 }
 
 .centered {
+  color: #2BBAE7;
+  font-size: 24px;
+  font-family: "Komikax";
```
Comment (rehandalal) [inline kitsune/sumo/static/sumo/less/mobile/main.less]: f < s

This should swap with font-size

**Comment by rehandalal** [comment]
Just those two small things and I'll merge this! Thanks!

**Comment by orvi2014** [comment]
I solve this issue. I think now we can merge it.

**Comment by rehandalal** [comment]
Perfect thanks!

**Comment by orvi2014** [comment]
Can you mark this as resolved in Bugzilla ? 

Thanks

**Comment by rehandalal** [comment]
@orvi2014 we don't mark things as resolved until they have been pushed to production. when that happens I (or whoever pushed it) will mark the bug as resolved.

**Comment by orvi2014** [comment]
oho I don't know that. Anyway thanks for helping.

**Pull request merged**